openmdao/drivers/tests/test_doe_driver.py

```diff
@@ -337,36 +318,36 @@ def test_csv_array(self):
         prob.run_driver()
         prob.cleanup()
 
-        expected = {
```

Comment:
This is the expected result shown in the figure. 

Which is equivalent to:
`pyDOE2.fullfact(levels=[2, 2, 2])
`

Which correctly results in:
```
array([[0., 0., 0., 0.],
       [1., 0., 0., 0.],
       [0., 1., 0., 0.],
       [1., 1., 0., 0.],
       [0., 0., 1., 0.],
       [1., 0., 1., 0.],
       [0., 1., 1., 0.],
       [1., 1., 1., 0.],
       [0., 0., 0., 1.],
       [1., 0., 0., 1.],
       [0., 1., 0., 1.],
       [1., 1., 0., 1.],
       [0., 0., 1., 1.],
       [1., 0., 1., 1.],
       [0., 1., 1., 1.],
       [1., 1., 1., 1.]])
```